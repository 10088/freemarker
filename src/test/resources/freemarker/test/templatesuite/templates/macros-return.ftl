<#macro m>m{<#return>}</#macro>
<@m/>

<#macro m><#if true>m{<#return>}</#if></#macro>
<@m/>

<#macro m><#if true>m{<#return></#if>}</#macro>
<@m/>

<#macro b>b{<#nested>}</#macro>
<#macro m><@b><#return></@></#macro>
<@m/>

<#macro b>b{<#nested>}</#macro>
<#macro m>m:<@b><#return></@></#macro>
<@m/>