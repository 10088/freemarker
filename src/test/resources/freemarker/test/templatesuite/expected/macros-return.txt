m{
m{
m{
b{
m:b{